proto/umee/oracle/v1/events.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package umee.oracle.v1;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/oracle/types";
+
+option (gogoproto.goproto_getters_all) = false;
+
+// EventDelegateFeedConsent is emitted on Msg/DelegateFeedConsent
+message EventDelegateFeedConsent {
+  // Operator bech32 address who delegates his feed consent
+  string operator = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // Delegate bech32 address
+  string delegate = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+// EventSetFxRate is emitted on Fx Rate update
+message EventSetFxRate {
+  // Asset denom
+  string denom = 1;
```

Comment:
```suggestion
  // Symbol denom
  string denom = 1;
```
This comment might help distinguish between `BaseDenom` and `SymbolDenom`

Reply:
what is symbol here? I don't see we use symbol in leverage.

Reply:
Symbol refers to the oracle acceptlist - leverage uses this as a key to get prices

Reply:
symbol sounds weird, especially in other comments: _convert to base token symbol denom_
I prefer to use `base denom`. We should simplify the language rather than adding more adjuncts

Reply:
Base denom is commonly referred to as the IBC denom, whereas this will look like “ATOM” or “JUNO”. Either way though I think we’re splitting hairs